features/fundraiser_can_create_a_campaign.feature

```diff
@@ -18,8 +23,23 @@ Feature: Create campaign
       Then I should see 'Bad Health'
       Then I should see 'I have a bad heath.'
 
-    Scenario: Create a campaign [Sad Path]
-      When I click 'Save Campaign' 
+    Scenario: Successfully create a campaign when logged in [Happy Path]
+      When I have logged in
+      And I click 'Create a Campaign'
+      And I fill in 'Goal' with '1000'
+      And I fill in 'Company name' with 'SAAB'
+      And I fill in 'Postal code' with '1234'
+      And I fill in 'Category' with 'Health'
+      And I fill in 'Title' with 'Bad Health'
+      And I fill in 'Description' with 'I have a bad heath.'
+      And I click 'Save Campaign' 
+      Then I should see 'Bad Health'
+      Then I should see 'I have a bad heath.'
+
+    Scenario: Create a campaign when logged in [Sad Path]
+      And I have logged in
```

Comment:
You mix up "then", "and" & "when" in this feature too much

Reply:
Cleaned this up now.